<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleFileResourceConfigDialog</class>
 <widget class="QWidget" name="SingleFileResourceConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>183</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0">
    <widget class="KTabWidget" name="ktabwidget">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Filename:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_Path</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KUrlRequester" name="kcfg_Path"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Display name:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_DisplayName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="kcfg_DisplayName"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ReadOnly">
         <property name="text">
          <string>Read only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_MonitorFile">
         <property name="text">
          <string>Enable file &amp;monitoring</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
